### DISCUSSION TITLE
snap client without i2s

### DISCUSSION THREAD

**Discussion by parabyte** [pschatzmann/arduino-snapclient#34] [Q&A] [answered]
I want to play around with homemade r2r dac as a educational project to do with my daughter, the board i wish to do this on, a stm32 blackpill board has no i2s, obviously for this experiment i do not need i2s, and compilation of the code is throwing this error

arduino-audio-tools/src/AudioI2S/I2SSTM32.h:5:10: fatal error: stm32-i2s.h: No such file or directory
    5 | #include "stm32-i2s.h"
      |          ^~~~~~~~~~~~~

**Comment by pschatzmann** [comment, marked as answer]
In the Arduino-Audio-Tools:
- Install the I2S library: https://github.com/pschatzmann/stm32-i2s
- Or deactivate the I2S functionality (USE_I2S) for the STM32 in AudioConfig.h (line 570)
